__tests__/app.test.js

```diff
@@ -103,6 +103,101 @@ describe("GET /api/articles/:article_id", () => {
       .expect(404)
       .then(({ body }) => {
         expect(body.msg).toBe('Object not found')
+        });
+    });
+})
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("takes an object and returns the updated article object", () => {
+    const input = { inc_votes: 3 };
+    const originArticle1 = {
+        article_id: 1,
+        title: 'Living in the shadow of a great man',
+        topic: 'mitch',
+        author: 'butter_bridge',
+        body: 'I find this existence challenging',
+        created_at: '2020-07-09T20:11:00.000Z',
+        votes: 100
+      }
+    return request(app)
+      .patch("/api/articles/1")
+      .send(input)
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.updatedArticle).toBe("object");
```

Comment:
This is not the most robust way to test for an object, in this case an array would also pass the test. If you must test for an object consider toBeInstanceOf(Object)